crates/next-custom-transforms/src/transforms/server_actions.rs

```diff
@@ -239,16 +246,15 @@ struct ServerActions<C: Comments> {
     rewrite_default_fn_expr_to_proxy_expr: Option<Box<Expr>>,
     rewrite_expr_to_proxy_expr: Option<Box<Expr>>,
 
-    exported_idents: Vec<(
-        /* ident */ Ident,
-        /* name */ Atom,
-        /* id */ Atom,
-    )>,
-
     annotations: Vec<Stmt>,
     extra_items: Vec<ModuleItem>,
     hoisted_extra_items: Vec<ModuleItem>,
-    export_actions: Vec<(/* name */ Atom, /* id */ Atom)>,
+
+    // A map of all server references (inline + exported): export_name -> reference_id
+    reference_ids_by_export_name: FxIndexMap<Atom, Atom>,
+
+    // A list of server references for originally exported server functions only.
+    server_reference_exports: Vec<ServerReferenceExport>,
```

Comment:
```suggestion
    /// A map of all server references (inline + exported): export_name -> reference_id
    reference_ids_by_export_name: FxIndexMap<Atom, Atom>,

    /// A list of server references for originally exported server functions only.
    server_reference_exports: Vec<ServerReferenceExport>,
```
then they will show in the LSP